### PR TITLE
Thumbnails fix

### PR DESCRIPTION
**Pull request by FredericaBernkastel**
Don't you think this is significantly better?

Thumbnails will fit container by width or height, without cropping and blurring, preserving aspect ratio, as in any other booru website.

### CODE REVIEW

**Comment by mserajnik** [comment]
Thanks, I'll try this tomorrow and provide feedback! 👍

One thing to mention is that thumbnails are generated by hydrus server and have a fixed size that can't be adjusted at the moment. I've asked the hydrus developer about it a few months ago and iirc he said that there might be options for that in the future if he plans to do more with the server.

If that becomes possible, it should be possible for users to adjust the size in hydrusrv (maybe not the actual displayed size in terms of browser window real estate, but it should be possible for them to trade smaller thumbnails for less bandwidth usage/delivery speed and vice versa).

File: src/styles/elements.scss [hunk 1]
```diff
@@ -131,15 +131,20 @@
...
+    align-items: center;
+    justify-content: center;
     transition: transform 200ms ease;
-    width: 100%;
+
+    img {
+      object-fit: contain;
+      height: 100%;
+    }
   }
```
Comment (mserajnik) [inline src/styles/elements.scss]: screenshot 2018-11-10 at 16 29 28

Please change the order of the properties to be alphabetical to get rid of the stylelint error:

```
.file-preview-image {
  align-items: center;
  background-color: $light;
  display: flex;
  height: 100%;
  justify-content: center;
  left: 0;
  position: absolute;
  top: 0;
  transition: transform 200ms ease;
  width: 100%;

  img {
    height: 100%;
    object-fit: contain;
  }
}
```

File: src/views/Files.vue [hunk 2]
```diff
@@ -24,11 +24,10 @@
...
           :key="index">
           <div class="file-preview">
-            <div
-              class="file-preview-image"
-              :style="{
-                backgroundImage: `url(${file.thumbnailUrl + mediaToken})`
-              }"></div>
+            <div class="file-preview-image">
+              <img 
+                :src="`${file.thumbnailUrl + mediaToken}`">
```
Comment (mserajnik) [inline src/views/Files.vue]: This can be in a single line and written in a simplified way:

```
<img :src="file.thumbnailUrl + mediaToken">
```

**Comment by FredericaBernkastel** [comment]
Thanks, i'm new to vue.

**Comment by mserajnik** [comment]
Merged into `develop` so it can be released together with the changes for #4.

Thanks!

**Pull request merged**